src/components/ErrorBox.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import Radium from 'radium';
+import {connect} from 'react-redux';
+import {styles} from './styles';
+
+export class ErrorBox extends React.Component {
+  render() {
+    switch(false) {
+      case typeof this.props.loadApiError === "undefined":
+        return (
+          <span style={styles.errorBox}>
+              {this.props.loadApiError.message}
+          </span>
+        );
+      case typeof this.props.runCodeError === "undefined":
+        return (
+          <span style={styles.errorBox}>
+              {this.props.runCodeError.message}
+          </span>
+        );
+      case typeof this.props.googleDriveError === "undefined":
+        return (
+          <span style={styles.errorBox}>
+              {this.props.googleDriveError.message}
+          </span>
+        );
+      default:
+        return null; //if no error recorded in state do nothing
+    }
+  }
+}
+
+ErrorBox.propTypes = {
+  loadApiError: React.PropTypes.string,
+  runCodeError: React.PropTypes.string,
+  googleDriveError: React.PropTypes.string,
+};
+
+export default connect(
+  state => ({
+    loadApiError: state.loadApi.error,
+    runCodeError: state.runCode.error,
+    googleDriveError: state.googleDrive.error,
+  })
```

Comment:
meanwhile, to pass in the error here, you should use a selector (which you will need to add to `selectors.js`). It could be something like:

```javascript
state => ({error: selectors.getAnyError(state)})
```

Then in `selectors.js` you would have something like:

```javascript
export function getAnyError(state) {
  return state.loadApi.error || state.runCode.error || state.googleDrive.error;
}
```

IMO, it's a little bit funky to store all these errors in different places, but probably not worth changing just yet.